src/backend/distributed/utils/query_utils.c

```diff
@@ -41,7 +41,8 @@ ExtractRangeTableList(Node *node, ExtractRangeTableWalkerContext *context)
 		RangeTblEntry *rangeTable = (RangeTblEntry *) node;
 
 		if (walkerMode == EXTRACT_ALL_ENTRIES ||
-			(rangeTable->rtekind == RTE_RELATION && rangeTable->relkind != RELKIND_VIEW))
+			(rangeTable->rtekind == RTE_RELATION && rangeTable->relkind != RELKIND_VIEW &&
```

Comment:
I think if you rebase onto master after #3110 is merged, can remove this change

Reply:
It is merged now